<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>460</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>37</y>
      <width>571</width>
      <height>275</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="laberlOperation">
           <property name="font">
            <font>
             <pointsize>28</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Результат:</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QLabel" name="labelResultNumerator">
             <property name="font">
              <font>
               <pointsize>28</pointsize>
              </font>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="labelDash">
             <property name="font">
              <font>
               <pointsize>18</pointsize>
              </font>
             </property>
             <property name="inputMethodHints">
              <set>Qt::ImhNoTextHandles</set>
             </property>
             <property name="text">
              <string>——————</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="labelResultDenominator">
             <property name="font">
              <font>
               <pointsize>28</pointsize>
              </font>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="spacing">
          <number>5</number>
         </property>
         <property name="sizeConstraint">
          <enum>QLayout::SetDefaultConstraint</enum>
         </property>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QTextEdit" name="fractionANumerator">
             <property name="maximumSize">
              <size>
               <width>145</width>
               <height>80</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>16</pointsize>
              </font>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTextEdit" name="fractionADenominator">
             <property name="maximumSize">
              <size>
               <width>145</width>
               <height>80</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>16</pointsize>
              </font>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="labelOperator">
           <property name="maximumSize">
            <size>
             <width>40</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>28</pointsize>
            </font>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QTextEdit" name="fractionBNumerator">
             <property name="maximumSize">
              <size>
               <width>145</width>
               <height>80</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>16</pointsize>
              </font>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTextEdit" name="fractionBDenominator">
             <property name="maximumSize">
              <size>
               <width>145</width>
               <height>80</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>16</pointsize>
              </font>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="buttonSum">
         <property name="text">
          <string>Сума</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonDifference">
         <property name="text">
          <string>Різниця</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonProduct">
         <property name="text">
          <string>Добуток</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonDivision">
         <property name="text">
          <string>Частка</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonPower">
         <property name="text">
          <string>До степеня</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonReduce">
         <property name="text">
          <string>Скоротити</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
